src/locale.js

```diff
@@ -1,13 +1,19 @@
+import { merge } from 'lodash/fp'
```

Comment:
you aren't doing any deep merging though, since  it's merging each namespace individually. I'd prefer to maintain that as well since `merge` is not the tiniest part of lodash